Tests/test_Tutorial.py

```diff
@@ -221,16 +227,21 @@ def funct(n, d, f):
             method = lambda x: None  # noqa: E731
             if f:
                 p = os.path.join(tutorial_base, f)
-                method.__doc__ = "%s\n\n>>> import os\n>>> os.chdir(%r)\n%s\n" \
-                    % (n, p, d)
+                method.__doc__ = "%s\n\n>>> import os\n>>> os.chdir(%r)\n%s\n" % (
+                    n,
+                    p,
+                    d,
+                )
```

Comment:
That looks horrible. How about an f-string here?

```python
method.__doc__ = f"{n}\n\n>>> import os\n>>> os.chdir({p!r})\n{d}\n" 
```

Reply:
Yes, looks better, updated, Thank you.

Reply:
My use of single letter variables here is also considered poor style, but this is simple black-compatible improvement. Thanks!